test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -244,17 +244,17 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
 '@
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'qywqqibm.q5p'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 186
186:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```
Test-Connection.TraceRoute.returns false without error if MaxHops is exceeded during -Traceroute -Quiet
```powershell
Missing an argument for parameter 'TimeoutSeconds'. Specify a parameter of type 'System.Int32' and try again.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Test-Connection.Tests.ps1: line 278
278:             Test-Connection 8.8.8.8 -Traceroute -MaxHops 2 -TimeoutSeconds -Quiet | Should -BeFalse

```